packages/stripSensitiveLiterals/src/index.ts

```diff
@@ -1,13 +1,33 @@
-import {
+import type {
+  ASTVisitor,
   DocumentNode,
   FloatValueNode,
   IntValueNode,
+  ListValueNode,
+  ObjectValueNode,
   StringValueNode,
-  visit,
 } from "graphql";
-// In the same spirit as the similarly named `hideLiterals` function, only
-// hide sensitive (string and numeric) literals.
-export function stripSensitiveLiterals(ast: DocumentNode): DocumentNode {
+import { visit } from "graphql";
+
+// Hide sensitive string and numeric literals. Optionally hide list and object literals with the option `hideListAndObjectLiterals: true`.
+export function stripSensitiveLiterals(
+  ast: DocumentNode,
+  options: { hideListAndObjectLiterals: boolean } = {
```

Comment:
technically this doesn't let you pass an empty option object...

Reply:
[4b90317](https://github.com/apollographql/apollo-utils/pull/92/commits/4b90317cadc4d507e828c3d9c3ddf64a89a67d74)